examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,151 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example reproduces figures from Lalor et al's mTRF toolbox in
+matlab [1]_. We will show how the :class:`mne.decoding.ReceptiveField` class
+can perform a similar function along with :mod:`sklearn`. We will fit a
+linear encoding model using the continuously-varying speech envelope to
+predict activity of a 128 channel EEG system.
+
+References
+----------
+.. [1] Crosse, M. J., Di Liberto, G. M., Bednar, A. & Lalor, E. C. (2016).
+       The Multivariate Temporal Response Function (mTRF) Toolbox:
+       A MATLAB Toolbox for Relating Neural Signals to Continuous Stimuli.
+       Frontiers in Human Neuroscience 10, 604. doi:10.3389/fnhum.2016.00604
+
+.. _figure 1: http://journal.frontiersin.org/article/10.3389/fnhum.2016.00604/full#F1
+.. _figure 2: http://journal.frontiersin.org/article/10.3389/fnhum.2016.00604/full#F2
+"""  # noqa: E501
+
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+# sphinx_gallery_thumbnail_number = 3
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.io import loadmat
+
+import mne
+from mne.decoding import ReceptiveField
+from sklearn.linear_model import Ridge
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import scale
+
+
+###############################################################################
+# Load the data from the publication
+# ----------------------------------
+#
+# First we will load the data collected in [1]_. In this experiment subjects
+# listened to natural speech. Raw EEG and the speech stimulus was collected.
+# We will load these below, downsampling the data in order to speed up
+# computation since we know that our features are primarily low-frequency in
+# nature. Then we'll visualize both the EEG and speech envelope.
+
+path = mne.datasets.mtrf.data_path()
+n_decim = 2
+data = loadmat(path + '/speech_data.mat')
```

Comment:
@agramfort says it works on Windows, so we now have a bit of a mix in our codebase